scala/private/phases/phase_compile.bzl

```diff
@@ -52,16 +52,6 @@ def phase_compile_library(ctx, p):
     )
     return _phase_compile_default(ctx, p, args)
 
-def phase_compile_library_for_plugin_bootstrapping(ctx, p):
-    args = struct(
-        unused_dependency_checker_ignored_targets = [
```

Comment:
how come we no longer need to ignore these targets?

Reply:
Since we turn off unused deps checker anyways (which now happens in phase_dependency_library_for_plugin_bootstrapping) there's no need to collect ignore targets at all from what I can see.

As a side note unused_deps_ignored_targets was originally part of the dependency_info struct, I moved it out in part because in phase_compile we made a decision to turn off unuse deps. But now that that logic is in phase_dependency again maybe it makes sense to have phase_dependency collect the ignored targets. (But there may have been other disadvantages of doing that, I can't recall)